manuscript.md

```diff
@@ -36,6 +40,45 @@
 
 ## What is a species interaction?
 
+Interactions between species can be manifested in a multitude of ways e.g. different 
+types [@Jordano2016ChaEco], strengths, or symmetry and are not necessarily restricted 
+between only a pair of species e.g indirect interactions [@Morales-Castilla2015InfBio]. The common thread 
+between these multiple forms of interactions is that *at least* one of the species is 
+affected by the interaction, this can be either positively or negatively [@Morales-Castilla2015InfBio]. Combine 
+this high 'diversity' of interaction types with the sheer number of potential 
+interactions that *could* occur in a community and it is understandable why attempting 
+to sample species interaction empirically is a considerable task. However, inferring 
+interactions through proxies and other frameworks could prove to be promising 
```

Comment:
> through proxies and other frameworks

needs a little more detail on what this means

Reply:
Urgh I want to link the change I just did to this convo but I can't find how..
Would something short like what I just changed work, or do you think we should elaborate a bit more on that matter?
Feel like the "how" would go in another section, what do you think?